gui/src/widgets/plotbufferpreviewer.cpp

```diff
@@ -95,7 +95,7 @@ void PlotBufferPreviewer::updateBufferPreviewer()
 
 	// Time interval that represents the captured data
 	QwtInterval dataInterval(std::min(m_bufferDataLimitMin, m_bufferDataLimitMax),
-				 std::fmax(m_bufferDataLimitMin, m_bufferDataLimitMax));
+				 std::max(m_bufferDataLimitMin, m_bufferDataLimitMax));
```

Comment:
did using fmax raise any issues while compiling or what's the reason for changing this?